src/MSI/Product.wxs

```diff
@@ -142,12 +142,12 @@ Licensed under the MIT license. See LICENSE file in the project root for full li
 
                  <Directory Id="BugTemplates" Name="BugTemplates">
```

Comment:
>BugTemplates [](start = 32, length = 12)

Should the install folder also be renamed? It seems like the component name of BugTemplateComp is also a candidate for renaming here...

Reply:
Changed to Issue. I assume that's what you meant